transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollEventLoopGroup.java

```diff
@@ -33,8 +33,8 @@
  * it only works on linux.
  */
 public final class EpollEventLoopGroup extends MultithreadEventLoopGroup {
-    static {
-        // Ensure JNI is initialized by the time this class is loaded by this time!
+    {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move the contents of this initializer to a standard constructor or to field initializers. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1171)